libcxx/include/__type_traits/datasizeof.h

```diff
@@ -47,7 +47,13 @@ struct __libcpp_datasizeof {
   };
 #endif
 
+  // Disable warnings related to the use of `offsetof` on non-standard layout or non-POD types
+  // TODO : Find a way to replace `offsetof` ?
```

Comment:
I don't think this TODO is actionable from the libc++ side of things currently, so I'd rather not add it. `offsetof` is the right tool for what we are trying to achieve - even if it's technically UB.